analysers/analyser_osmosis_powerline.py

```diff
@@ -681,50 +681,71 @@ def __init__(self, config, logger = None):
-Add a transformer using `power=transformer` (standalone transformers) or `power=pole + transformer=*` (pole mounted transformers).'''))
+If yes, add a substation (`power=substation`) or transformer (`power=transformer`) on this node (standalone transformers)
+or `power=pole + transformer=*` (pole mounted transformers).'''),
+            trap = T_(
+'''Some lines may share the same support without connection.
+Have a look to `line_management=termination` or `line_management=cross` for such sitations.'''))
         self.classs[4] = self.def_class(item = 7040, level = 3, tags = ['power', 'fix:imagery'],
             title = T_('Non power node on power way'),
             detail = T_(
-'''Power lines can only form a straight line between supports and therefore shouldn't
+'''Power lines can only form a straight line between their supports and therefore shouldn't
 have additional nodes that aren't tagged as a `power` feature.'''),
             fix = T_(
 '''If this node is a tower or pole, use the tag `power=tower` or
 `power=pole`. Otherwise remove it.'''))
         self.classs_change[5] = self.def_class(item = 7040, level = 3, tags = ['power', 'fix:imagery'],
             title = T_('Missing power tower or pole'),
             detail = T_(
-'''Based on the statistical frequency of the poles on this power line,
-there's likely an unmapped pole nearby.'''))
+'''Based on the statistical frequency of the supports on this power line,
+there's likely an unmapped pole nearby.'''),
+            fix = T_(
+'''Seek for possibly missing support in the surroundings and add it to the line.'''))
         self.classs[7] = self.def_class(item = 7040, level = 3, tags = ['power', 'fix:chair'],
-            title = T_('Unmatched voltage of line on substation'))
+            title = T_('Unmatched voltage of line on substation'),
+            detail = T_(
+'''A line terminates near of a substation with inconsistent voltages between them.'''),
+            fix = T_(
+'''Continue the line towards its actual endpoint, check voltages on line and on substation and adjust what is required to better reflect what is seen on ground.''')),
         self.classs[8] = self.def_class(item = 7040, level = 3, tags = ['power', 'fix:chair'],
-            title = T_('Power support line management suggestion'))
-
+            title = T_('Power support line management suggestion'),
+            detail = T_(
+'''[line_management](https://wiki.openstreetmap.org/wiki/Key:line_management) documents particular topologies occuring on power lines nodes. Mind looking at values documentation.
+Suggestions are computed by looking at existing OpenStreetMap power lines and only supports some combinations.
+It's an encouragement to check and confirm the structure of the power grid on most relevant towers, particularly when proposed `line_management` differs from existing value.
+Proposed values are to be adopted when conform to ground according to your knowledge, aerial imagery or survey.'''),
```

Comment:
Not sure what this means?
Unless it's a generic "tagging should confirm to ground truth"-statement (in which case it's unnecessary)